index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Project</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   <header>
+     <div class="welcome">
+        <h1>Hi there</h1>
+     </div>
+     <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+     </div>
+   </header> 
+   <section class="dashboard">
+    <h1 class="name">Abednego Kibitok</h1>
+    <h2 class="title">Software Developer</h2>
+    <p class="pa">
```

Comment:
- [ ] Your design does not follow that of the Figma template. 

Figma Design | Your Design
---- | ----
![image](https://user-images.githubusercontent.com/84156714/212360409-b3080e76-d737-48c1-9a83-2e0acdd3447c.png) | ![image](https://user-images.githubusercontent.com/84156714/212360483-94d16be6-dd74-433d-b5ad-495a73393817.png)

The Figma Design does not contain this portion as you have in your design. Please remove it.
![image](https://user-images.githubusercontent.com/84156714/212360773-bb29021c-80d9-48d7-b7fc-830adc279b2e.png)